spec/features/merchant/orders/show_spec.rb

```diff
@@ -58,4 +60,30 @@
     expect(page).to have_content("#{@item_order_1.quantity}")
   end
 
+  it "if order is unfulfilled and the desired quantity is equal to or less than my inventory, I see fulfill button" do
+    visit "/merchant/orders/#{@order.id}"
+    within "#item-#{@dog_bone.id}" do
+      click_on "fulfill item"
+    end
+
+    expect(current_path).to eq("/merchant/orders/#{@order.id}")
+    @dog_bone.reload
```

Comment:
Have you guys used reload? This test wouldn't pass even though the inventory seemed to be updating as expected. THEN I found this on-line and tried it and BOOM!

Reply:
Yeah reload is great and this is the perfect use case for it!